src/pharmpy/plugins/nlmixr/model.py

```diff
@@ -253,9 +254,41 @@ def execute_model(model):
     # and calling other R version.
     newenv['R_LIBS_USERS'] = ''
     newenv['R_LIBS_SITE'] = ''
-    subprocess.run([rpath, path / (model.name + '.R')], env=newenv)
+
+    stdout = path / 'stdout'
+    stderr = path / 'stderr'
+
+    args = [rpath, path / (model.name + '.R')]
+
+    with open(stdout, "wb") as out, open(stderr, "wb") as err:
```

Comment:
Do we need to open these? I think that `sys.stdout` and `sys.stderr` are always open.

Reply:
These are not `sys.stdout`/`sys.stdin`. These are temporary file paths. Maybe naming is confusing?